### DISCUSSION TITLE
Full flaps

### DISCUSSION THREAD

**Discussion by rkkpmptk** [flybywiresim/aircraft#2309] [Help]
when i  pause the sim then resume the flaps go to full flpas, autothrottle diengage and full spoilers

**Comment by WelshSimmer** [comment]
Try use Esc to pause and not active pause

**Reply by rkkpmptk** [reply]
yes this is what im doing

**Comment by nyhmbo** [comment]
do you have any other mods installed? if so, try only with the a32nx mod in your community folder.

**Reply by rkkpmptk** [reply]
i do have livries, and sceneries, but i tried reinstall and it didnt work

**Reply by nyhmbo** [reply]
you might want to check this out: https://forums.flightsimulator.com/t/flaps-and-spoilers-fully-extend-after-pressing-esc-escape-during-flight/312467/20
looks like its a controller/key assignment issue

**Comment by Saschl** [comment]
Do you use the Thrustmaster TCA quadrant? If yes, remove the mappings for spoilers and flaps from the menu

**Comment by sai-barath** [comment]
Is it possible that you’re using the same keybinds to pause and extend flaps?

**Comment by derl30n** [comment]
Are you using Thrustmeter TCA? If so unbind any non existed buttons on your hardware.
